docs/docs/contribution/contribution-short-guide.md

```diff
@@ -97,6 +97,23 @@ npm run test:dev
 2. Optional: Design - If you're doing something that is not straightforward, share your high-level approach to this within the issue
 3. PR - Once you're done, run the tests locally then PR to main. Ensure all checks pass. If you introduced a new feature - Update the docs
 
+## Commit standards
+
+ Commit messages must follow [conventional commit guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary) (this is enforced by commitlint so don't worry about making a mistake).
+ You must choose a commit message prefix from [these options](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum).
+ 
```

Comment:
Since this is the short guide, I would just leave one sentence here like:

Commit messages must follow [conventional commit guidelines (https://www.conventionalcommits.org/en/v1.0.0/#summary), just type 'npm run commit' or read the full guide for more info